client.lua

```diff
@@ -149,10 +151,10 @@ function PolyZone:draw()
   end
 
   if #points > 2 then
-    local firstPoint = self:TransformPoint(points[1])
-    local lastPoint = self:TransformPoint(points[#points])
-    DrawLine(firstPoint.x, firstPoint.y, maxZ, lastPoint.x, lastPoint.y, maxZ, oR, oG, oB, 184)
-    _drawWall(firstPoint, lastPoint, minZ, maxZ, wR, wG, wB, 48)
+      local firstPoint = self:TransformPoint(points[1])
+      local lastPoint = self:TransformPoint(points[#points])
+      DrawLine(firstPoint.x, firstPoint.y, maxZ, lastPoint.x, lastPoint.y, maxZ, oR, oG, oB, 184)
+      _drawWall(firstPoint, lastPoint, minZ, maxZ, wR, wG, wB, 48)
```

Comment:
Looks like this got left with extra indentation. Drop these back two spaces and then we should be good to merge!